packages/swapper/src/swappers/cow/CowSwapper.ts

```diff
@@ -76,6 +78,12 @@ export class CowSwapper implements Swapper<KnownChainIds.EthereumMainnet> {
     return cowApproveInfinite(this.deps, args)
   }
 
+  async approveAmount(): Promise<string> {
```

Comment:
- This never returns a string on the unimplemented implementations.
- The implemented ones are also not *guaranteed* to return a string, they can throw and thus return `void`.
Let's change the type definition of the implemented ones to `string | void`, and the one from the unimplemented ones to `void`?

Reply:
More accurately, this function would be assigned a return type of never (which does satisfy the swapper interface), though we don't really interact with specific swapper types in the client.

Considering this is an async function that should be wrapped in a try/catch, I don't see too big of a problem with this current implementation and it follows how we have handled unimplemented function up until this point as far as I am aware.